hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/helpers/ChunkUtils.java

```diff
@@ -180,6 +183,15 @@ private static long writeDataToChannel(FileChannel channel, ChunkBuffer data,
     }
   }
 
+  public static ChunkBuffer readData(long len, int bufferCapacity,
+      CheckedConsumer<ByteBuffer[], StorageContainerException> readMethod)
+      throws StorageContainerException {
+    final ByteBuffer[] buffers = BufferUtils.assignByteBuffers(len,
+        bufferCapacity);
+    readMethod.accept(buffers);
+    return ChunkBuffer.wrap(Lists.newArrayList(buffers));
```

Comment:
`ChunkBuffer` doesn't change the list's size, and it creates a defensive copy anyway.  So I think we could use `Arrays.asList()` here.